openapi.yaml

```diff
@@ -1,190 +1,480 @@
-  PaginationLinks:
+  OwnerResource:
+    required: [id, type, attributes]
     properties:
-      first:
-        type: string
-        format: url
-        description: The first page of data
-      last:
-        type: string
-        format: url
-        description: The last page of data
-      prev:
+      id: &ownerId
         type: string
-        format: url
-        description: The previous page of data
-      next:
+        example: '0001'
+        description: A unique ID of a park owner
+      type: &ownerType
         type: string
-        format: url
-        description: The next page of data
-  PetResource:
+        description: type of object, should be owner
+        enum: [owner]
+      attributes: &ownerAttributes
+        required: [ownerName]
+        properties:
+          ownerName: &ownerName
+            type: string
+            description: Name of park owner / organization
+            example: City of Corvallis
+  PostOwnerModel:
+    required: [data]
+    properties:
+      data:
+        required: [type, attributes]
+        properties:
+          type: *ownerType
+          attributes: *ownerAttributes
+  PatchOwnerModel:
+    required: [data]
+    properties:
+      data:
+        required: [id, type]
+        properties:
+          id: *ownerId
+          type: *ownerType
+          attributes:
+            properties:
+              ownerName: *ownerName
+  ParkResource:
+    required: [id, type, attributes, relationships]
     properties:
       id:
         type: string
-        description: A unique ID of pet
-      type:
+        example: '0001'
+        description: A unique ID of park
+      type: &parkType
         type: string
-        enum: [pet]
-      links:
-        $ref: '#/definitions/SelfLink'
-      attributes:
+        description: type of object, should be park
+        enum: [park]
+      attributes: &parkAttributes
+        required: [name, location, amenities]
         properties:
-          name:
+          name: &parkName
             type: string
-            description: Name of pet
-            example: Hedwig
+            description: Name of Park
+            example: Avery Park
+          location:
+            type: object
+            required: [streetAddress, city, state, zip, latitude, longitude]
+            properties:
+              streetAddress: &streetAddress
+                type: string
+                description: Park address
+                example: '1200 SW Avery Park Dr.'
+              city: &city
+                type: string
+                description: City where the park is located
+                example: Corvallis
+              state: &state
+                type: string
+                description: Abbreviation of state where park is located
+                pattern: &states '^(?:(A[KLRZ]|C[AOT]|D[CE]|FL|GA|HI|I[ADLN]|K[SY]|LA|M[ADEINOST]|N[CDEHJMVY]|O[HKR]|P[AR]|RI|S[CD]|T[NX]|UT|V[AIT]|W[AIVY]))$'
```

Comment:
It looks like this pattern is defined above in a query parameter and has the same value. Could you store it in the first definition so you can reference it here without redefining it?